modules/nodes-containers-dev-fuse-configuring.adoc

```diff
@@ -11,56 +11,66 @@ As an alternative to the virtual filesystem, you can configure the `/dev/fuse` d
 .Procedure
 
 . Create the pod.
-+
+
 [source,terminal]
 ----
-$ oc exec -ti no-priv -- /bin/bash
+oc apply -f podman-pod.yaml
 ----
-+
-[source,terminal]
+
+[IMPORTANT]
+====
+Ensure that the pod is created and running commands before continuing this procedure. Here is an example configuration similar to what you should see.
+
+[source,yaml]
 ----
-$ cat >> Dockerfile <<EOF
-FROM registry.access.redhat.com/ubi9
-EOF
+apiVersion: v1
+kind: Pod
+Metadata:
+name: podman-pod
+Annotations:
+io.kubernetes.cri-o.Devices: "/dev/fuse"
+Spec:
+containers:
+- name: podman-container
+image: quay.io/podman/stable
+       Args:
+- sleep
+- "1000000"
+securityContext:
+runAsUser: 1000
```

Comment:
Oh is this podman-pod.yaml? 
Maybe step 1 should be 

. Create a YAML file similar to the following example pod spec:

```
apiVersion: v1
kind: Pod
Metadata:
name: podman-pod
Annotations:
io.kubernetes.cri-o.Devices: "/dev/fuse"
Spec:
containers:
- name: podman-container
image: quay.io/podman/stable
       Args:
- sleep
- "1000000"
securityContext:
runAsUser: 1000
```
. Create the pod by running the following command:
+
[source,terminal]
----
$ oc create -f <file_name>.yaml
----
+
[NOTE]
====
Ensure that the pod is created and running commands before continuing this procedure. 
====